lightning/src/lib.rs

```diff
@@ -17,6 +17,29 @@
 //! figure out how best to make networking happen/timers fire/things get written to disk/keys get
 //! generated/etc. This makes it a good candidate for tight integration into an existing wallet
 //! instead of having a rather-separate lightning appendage to a wallet.
+//! 
+//! Available features are: 
+//! 
+//! 1. Skip logging of messages at levels below the given log level:
```

Comment:
Maybe we can switch the order here, so that the arguably more important (default) features are described up top, and the logging/debugging things are listed beneath?